### ISSUE TITLE
AttributeError: 'Client' object has no attribute 'open'

### ISSUE THREAD

**Issue by gonesbuyo** [graknlabs/grakn-python#30]
I get this error in Grakn 1.2 when I try to run the example, I tried with Grakn 1.0 but I obtain the same error.
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-16-cc385f197a04> in <module>()
----> 1 with client.open() as tx:
        2     tx.execute('define person sub entity;')

AttributeError: 'Client' object has no attribute 'open'
```

I'm running the `basic-genealogy.gql` example file, previously loaded from terminal with the appropriate command (`./graql console -k genealogy -f ./examples/basic-genealogy.gql`):
```
import grakn
client = grakn.Client(uri='localhost:4567', keyspace='genealogy')
with client.open() as tx:
    tx.execute('define person sub entity;')
```

**Comment by sjpritchard** [comment]
I also have this error. Any update?

**Comment by cyclecycle** [comment]
I also have this error.

**Comment by flyingsilverfin** [comment]
Hi all - 
@cyclecycle  is correct in noticing the pip package and git repository are in different states. You should be using `client.execute("...")` rather than calling `client.open()`. The current git repository is not in a functional state and is being rewritten as we speak -- to be released with Grakn 1.3 end of this week or early next week!

**Comment by flyingsilverfin** [comment]
@cyclecycle Are you still having issues with the JSONDecodeError?
